modules/cudf/src/data_frame.ts

```diff
@@ -847,6 +848,63 @@ export class DataFrame<T extends TypeMap = any> {
     }
   }
 
+  /**
+   * Perform a math operation on a column that is of type NumericSeries
+   * @ignore
+   */
+  _performMathOp<R extends DataType>(columnName: string, mathOp: () => Series<R>): Series<R> {
+    if (this.get(columnName) instanceof NumericSeries) { return mathOp(); }
+    return Series.new(this._accessor.get(columnName));
+  }
```

Comment:
Could this be a standalone local utility function instead of a public method on `DataFrame`?